### DISCUSSION TITLE
Cannot open VcXsrv (Peacock) Windows 11

### DISCUSSION THREAD

**Discussion by Erikad1212** [idaholab/moose#22507] [Q&A Getting Started] [answered]
I have recently downloaded MOOSE using the Windows 10 instructions, however, I am technically on Windows 11. I am unable to display VcXsrv to use peacock but I can run the examples provided in moose. I have ran through the downloading steps again to ensure I did not miss a step. Does anyone have any advice?

**Comment by GiudGiud** [comment]
Hello

Did you do everything in the documentation:
https://mooseframework.inl.gov/getting_started/installation/windows.html#f21a2a8e-8f48-4c31-bc33-f52b73e4f1a4

- Launched VcXsrv before WSL
- Disabled access control
- Allowed network connections VcXsrv
Guillaume

**Reply by milljm** [reply, marked as answer]
Windows 11 provides native GL support. There is no need to install VcXsrv (Ref a recent discussion: https://github.com/idaholab/moose/discussions/22718#discussioncomment-4276193). None of us here have a Windows 11 machine to test on, this was something I discovered via Microsoft's help topics).

However, you may still find some useful information within that post...
